tests/Export-DbaDacPackage.Tests.ps1

```diff
@@ -4,32 +4,52 @@ Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
 
 Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
     Context "Validate parameters" {
-        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object {$_ -notin ('whatif', 'confirm')}
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object { $_ -notin ('whatif', 'confirm') }
         [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'Database', 'ExcludeDatabase', 'AllUserDatabases', 'Path', 'FilePath', 'DacOption', 'ExtendedParameters', 'ExtendedProperties', 'Type', 'Table', 'EnableException'
         $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
         It "Should only contain our specific parameters" {
-            (@(Compare-Object -ReferenceObject ($knownParameters | Where-Object {$_}) -DifferenceObject $params).Count ) | Should Be 0
+            (@(Compare-Object -ReferenceObject ($knownParameters | Where-Object { $_ }) -DifferenceObject $params).Count ) | Should Be 0
         }
     }
 }
 
 Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     BeforeAll {
+        $server = $script:instance1
         $dbname = "dbatoolsci_exportdacpac"
         try {
-            $server = Connect-DbaInstance -SqlInstance $script:instance1
+            $server = Connect-DbaInstance -SqlInstance $server
             $null = $server.Query("Create Database [$dbname]")
-            $db = Get-DbaDatabase -SqlInstance $script:instance1 -Database $dbname
+            $db = Get-DbaDatabase -SqlInstance $server -Database $dbname
             $null = $db.Query("CREATE TABLE dbo.example (id int, PRIMARY KEY (id));
             INSERT dbo.example
             SELECT top 100 object_id
             FROM sys.objects")
         } catch { } # No idea why appveyor can't handle this
 
         $testFolder = 'C:\Temp\dacpacs'
+
+        $dbName2 = "dbatoolsci:2"
+        $dbName2Escaped = "dbatoolsci$2"
+
+        $null = New-DbaDatabase -SqlInstance $server -Name $dbName2
     }
     AfterAll {
-        Remove-DbaDatabase -SqlInstance $script:instance1 -Database $dbname -Confirm:$false
+        Remove-DbaDatabase -SqlInstance $server -Database $dbname, $dbName2 -Confirm:$false
+    }
+
+    Context "Bug 7038" {
```

Comment:
Please add a statement. A comment can be added to link to the issue number if desired.